tekton/mario-bot/mario-github-comment.yaml

```diff
@@ -48,35 +48,38 @@ spec:
     description: The name of the image build PipelineRun for the logs link
   resourcetemplates:
   - apiVersion: tekton.dev/v1beta1
-    kind: CustomRun
+    kind: PipelineRun
     metadata:
       generateName: mario-comment-github-
     spec:
-      customRef:
-        apiVersion: tekton.dev/v1alpha1
-        kind: PipelineToTaskRun
-        name: mario-comment-github
+      pipelineSpec:
```

Comment:
I'm not entirely sure that this will run correctly, but it will deploy, and it's Mario, so eh. =)